networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-azure.adoc

```diff
@@ -0,0 +1,135 @@
+[id="nw-control-dns-records-public-hosted-zone-azure_{context}"]
+= Controlling DNS records on an public hosted zone for Azure by using Red Hat External DNS Operator
```

Comment:
Creating DNS records on an public hosted zone for Azure by using Red Hat External DNS Operator